client/src/test/java/TestRunner.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+#run GeneralSMATest
+javac -cp "../../../build/libs/clientDesktop-0.0.1-alpha.jar" GeneralSMATest.java
+
+java -cp "../../../build/libs/clientDesktop-0.0.1-alpha.jar" GeneralSMATest.java
```

Comment:
Could you try to avoid copy-pasting `../../../build/libs/clientDesktop-0.0.1-alpha.jar`?

Reply:
The path is now saved in a variable. ef3475471e6adc8644212c105460a26493c4c4b3